tbx/static_src/sass/components/_streamfield.scss

```diff
@@ -0,0 +1,62 @@
+@use "config" as *;
+
+.streamfield {
+    &__title {
```

Comment:
Can you add some brief info to the documentation about how blocks are positioned in the grid?

Reply:
Added in https://github.com/torchbox/torchbox.com/pull/33/commits/69ae1f3a061566dd6c3f8a3146ff57efb32dda27